test/athens/common_events/linkmaker_test.cljc

```diff
@@ -368,23 +370,28 @@
         (let [{testing-block-1-eid :db/id}      (get-block testing-block-1-uid)
               {target-page-1-refs :block/_refs} (get-page target-page-1-uid)
               {target-page-2-refs :block/_refs} (get-page target-page-2-uid)
-              split-block-event                 (common-events/build-split-block-event
-                                                  testing-block-1-uid
-                                                  testing-block-1-string
-                                                  split-index
-                                                  testing-block-2-uid)
-              split-block-tx                    (resolver/resolve-event-to-tx @@fixture/connection split-block-event)]
+              split-block-op                    (graph-ops/build-block-split-op @@fixture/connection
+                                                                                {:old-block-uid testing-block-1-uid
+                                                                                 :new-block-uid testing-block-2-uid
+                                                                                 :string        testing-block-1-string
+                                                                                 :index         split-index
+                                                                                 :relation      :after})
+              block-split-atomics               (graph-ops/extract-atomics split-block-op)]
           ;; assert that target pages has no `:block/refs` to start with
           (t/is (= [{:db/id testing-block-1-eid}] target-page-1-refs))
           (t/is (= [{:db/id testing-block-1-eid}] target-page-2-refs))
 
           ;; apply split-block
-          (transact-with-linkmaker split-block-tx)
+          (doseq [atomic-op block-split-atomics
+                  :let      [atomic-txs (atomic-resolver/resolve-atomic-op-to-tx @@fixture/connection atomic-op)]]
+            (transact-with-linkmaker atomic-txs))
+
           (let [{testing-block-2-eid :db/id}      (get-block testing-block-2-uid)
                 {target-page-1-refs :block/_refs} (get-page target-page-1-uid)
                 {target-page-2-refs :block/_refs} (get-page target-page-2-uid)]
             ;; assert that we do have new ref
             (t/is (= [{:db/id testing-block-1-eid}] target-page-1-refs))
+            ;; TODO (RTC): this test is failing right now
```

Comment:
is it really?

Reply:
done, and also fixed how block/remove didn't rename title strings